utils/web_ui.py

```
@@ -16,6 +16,10 @@
 from pathlib import Path
 from openai import OpenAI
 import ftfy
+import subprocess
+import json
```

Ruff (0.11.9)

`json` imported but unused

Remove unused import: `json`

(F401)